components/brave_shields/browser/brave_shields_util_unittest.cc

```diff
@@ -168,6 +171,90 @@ TEST_F(BraveShieldsUtilTest, SetBraveShieldsEnabled_ForOrigin) {
   setting = map->GetContentSetting(GURL(), GURL(),
                                    ContentSettingsType::BRAVE_SHIELDS);
   EXPECT_EQ(CONTENT_SETTING_ALLOW, setting);
+
+  GURL blocked("http://blocked.com");
+  GURL allowed("http://allowed.com");
+  // Set allowed as blocked for initial value.
```

Comment:
This is very confusing. I don't think using "allowed" and "blocked" for domains is a good idea. Also we should be using enabled/disabled as well here for everything except the actual content setting value itself

Reply:
renamed

Reply:
sorry, I didn't mean change these to enable/disable, my point is that using any variation of block/allow/enable/disable is confusing because you are changing the content setting values for them. Something generic like host1.com, host2.com is better here I think.

Reply:
renamed, thanks